### ISSUE TITLE
Correct problems ends with (void-function popup-make-item)

### ISSUE THREAD

**Issue by kleewho** [senny/emacs-eclim#183]
After merging https://github.com/senny/emacs-eclim/pull/169 when calling `eclim-problems-correct` I get the `(void-function popup-make-item)`. I believe the it's because it's not mentioned in `eclim-pkg.el` nor required in `eclim-problems.el`. The solution most probably would be to add it to package definition:

```elisp
(define-package "emacs-eclim" "0.3" 
  "An interface to the Eclipse IDE." 
  '((json "1.2") 
    (s "1.9.0")
    (popup "v0.5.2")))
```

add this to `eclim-problems.el`

```elisp
(require `popup)
```

I think the reason why it's working for others is that they're using `auto-complete` instead of `company` and `auto-complete` requires `popup.el`.

It would be good to list `popup.el` as a dependency also in `README.md`

**Comment by jfeltz** [comment]
I just wanted to confirm both this problem and the ```(require `popup)``` part of the fix for master and emacs-24.4.2. Thanks for posting this.

**Comment by kleewho** [comment]
@jchochli You can close it. I don't know why my commit message didn't close it automatically

**Comment by kleewho** [comment]
Actually I can close it myself. It was fixed by #185

**Comment by jchochli** [comment]
> Actually I can close it myself. It was fixed by #185

@kleewho Nice!
